src/components/common/Pagination.tsx

```diff
@@ -88,6 +88,15 @@ const Pagination: FC<PaginationProps> = ({ viewQuerySettings, setViewQuerySettin
                         />
                     </div>
                 )}
+                {!showControls && (
```

Comment:
We should be able to just use the same logic as above and move the `showControls` check to wrap the button elements.

Reply:
Could do it like this:
![image](https://user-images.githubusercontent.com/1945295/207431180-8ee58079-91a2-44ce-aed4-ce1be0dc695d.png)